packages/fxa-settings/src/pages/ResetPassword/index.test.tsx

```diff
@@ -80,7 +103,7 @@ describe('PageResetPassword', () => {
   });
 
   it('renders a custom service name in the header when it is provided', () => {
-    render(<ResetPassword serviceName={MozServices.MozillaVPN} />);
+    render(<ResetPassword />, 'service=Mozilla VPN');
```

Comment:
Discovered later: I was only partially right, this can also be the oauth client ID; from what I can tell we've either got the  `client_id` param (before password reset link), or it's appended as `service` on the PW reset link which can also be `'sync'`.